plugin/src/main/java/com/playmonumenta/epicstructures/managers/RespawningStructure.java

```diff
@@ -251,12 +268,42 @@ public void run() {
 						if (mSpawnerBreakTrigger != null) {
 							mSpawnerBreakTrigger.resetCount();
 						}
+
+						mForcedRespawn = false;
+						mConquered = false;
 					}
 				}.runTask(mPlugin);
 			}
 		}.runTaskAsynchronously(mPlugin);
 	}
 
+	public void forcedRespawn(Player player) {
+		if (player.hasMetadata("ForceReset")) {
+			player.removeMetadata("ForceReset", mPlugin);
+		} else {
+			player.sendMessage(ChatColor.RED + "" + ChatColor.BOLD + "Waited too long! Try again.");
+			return;
+		}
+
+		if (mForcedRespawn) {
+			player.sendMessage(ChatColor.RED + "" + ChatColor.BOLD + "This Point of Interest is already forced to respawn!");
+		} else {
+			if (mSpawnerBreakTrigger.getSpawnerRatio() < 0.2) {
```

Comment:
This actually shouldn't be 0.2. The structures are all set so their spawner count is the number to break to conquer. The actual number of spawners contained within the POI is higher than that. So same logic, just compare against 0, not 0.2.